### ISSUE TITLE
Is email functionality planned?

### ISSUE THREAD

**Issue by Mrs-Feathers** [hcengineering/platform#5831]
I have my own working email server to send my own email notifications, all i need to do is plug in the SMTP login info and it works. if its because you can't use your own servers then why not let us use our own?

i would love to use this tool but no email notifications of tasks and changes is a deal breaker :(

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NjZlMTQ2NDJkOTA4MTUzNTY4M2NlMjAiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.veuvI1GzzOxD-3XAvOtAtCjt1Ig9raTTfK22DsOVaZE">Huly&reg;: <b>UBERF-7288</b></a></sub>

**Comment by BykhovDenis** [comment]
We have email notifications for the most important events. If you have any problem with it, could you please provide additional information (your workspace, email)

**Comment by aplatoff** [comment]
Hey @Mrs-Feathers,

We're reworking the notifications layer so users will be able to configure how notifications are delivered and which channels are used (SMTP, Telegram, Slack, push notifications, etc.). We hope to roll this out in 2-4 weeks. Until then, we do not have an option for you to use your own SMTP server.

Thank you for expressing your interest in using your own SMTP servers :)

**Comment by davidkobielski** [comment]
@aplatoff Hey,
is there know any date when this option will be available?
For now using selfhosted version is not easy and fun without this...
for example there is no way to login on another device, because it need code from email...

**Comment by Kantets** [comment]
Yes, I would really like to get this functionality, because going through verification and fighting with Amazon support to use mail is a pain, because of which it is impossible to fully enjoy the product. And there is no way to finish it, because the project is mostly not open source, the necessary modules and packages are in private repositories

**Comment by BykhovDenis** [comment]
@Kantets What do you mean? All project source is open, email service source placed here [https://github.com/hcengineering/platform/tree/develop/services/ses/pod-ses/src](https://github.com/hcengineering/platform/tree/develop/services/ses/pod-ses/src)

**Comment by zrbbv** [comment]
> @Kantets What do you mean? All project source is open, email service source placed here https://github.com/hcengineering/platform/tree/develop/services/ses/pod-ses/src

Do we have functionality to use other transactional email providers aside from SES? Like Mailgun, Mailtrap, Postmark, SMTP2GO, etc? 

This would be a great feature to add!
